vulnerabilities/importers/istio.py

```diff
@@ -155,23 +157,23 @@ def process_file(self, path):
             )
 
             safe_purls_golang = {
-                PackageURL(type="golang", name="istio", version=version)
+                PackageURL(type="istio", name="golang", version=version)
```

Comment:
I meant swap the order not values of the type and name. It's wrong everywhere now :)

Reply:
I think I messed up with the awk command. Glad you noticed.